### PR TITLE
docs: free and open terminology

### PR DESCRIPTION
**Pull request by bmorelli25**

### CODE REVIEW

File: docs/guide/apm-doc-directory.asciidoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ image::images/apm-architecture-cloud.png[Architecture of Elastic APM]
 [float]
 ==== APM Agents
 
-APM agents are open source libraries written in the same language as your service.
+APM agents are free and open libraries written in the same language as your service.
```
Comment (bmorelli25) [inline docs/guide/apm-doc-directory.asciidoc:11]:
```suggestion
APM agents are open source libraries written in the same language as your service.
```

File: docs/apm-package/apm-integration.asciidoc [hunk 2]
```diff
@@ -70,7 +70,7 @@ Agents::
 unified agent that you can deploy to hosts or containers to collect data and send it to the {stack}.
 Behind the scenes, {agent} runs APM Server to listen for `apm` data.
 +
-{apm-overview-ref-v}/components.html[**APM agents**] are open source libraries written in the same language as your service.
+{apm-overview-ref-v}/components.html[**APM agents**] are free and open libraries written in the same language as your service.
```
Comment (simitt) [inline docs/apm-package/apm-integration.asciidoc:73]: Actually, the APM agent libraries are `Apache 2.0` license

Comment (bmorelli25) [reply docs/apm-package/apm-integration.asciidoc:73]:
```suggestion
{apm-overview-ref-v}/components.html[**APM agents**] are open source libraries written in the same language as your service.
```

**Review by simitt — APPROVED** [review]
LGTM - thanks!

**Pull request merged**